src/interfaces/editorSettings.ts

```diff
@@ -0,0 +1,4 @@
+export interface EditorSettings {
```

Comment:
Maybe add a comment here, tracking the vscode issue, so we can remove this later on with confidence if/when they copy this interface.